packages/gatsby-theme-docs/src/components/card.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@emotion/css';
+import { designSystem } from '@commercetools-docs/ui-kit';
+import markdownFragmentToReact from '../utils/markdown-fragment-to-react';
+import Link from './link';
+
+const flatStyle = css`
+  border: 1px solid ${designSystem.colors.light.borderSecondary};
+`;
+const clickableStyle = css`
+  cursor: pointer;
+  box-shadow: ${designSystem.tokens.shadowForClickableCard};
+  :hover {
+    box-shadow: ${designSystem.tokens.shadowForClickableCardOnHover};
+  }
+  transition: box-shadow 0.1s ease-in-out;
+`;
+const Container = styled.li`
+  ${(props) => (props.clickable ? clickableStyle : flatStyle)};
+  background-color: ${designSystem.colors.light.surfacePrimary};
+  padding: ${designSystem.dimensions.spacings.m};
+  border-radius: ${designSystem.tokens.borderRadiusForCard};
+`;
+const Icon = styled.div`
+  width: ${designSystem.dimensions.spacings.big};
+  min-width: ${designSystem.dimensions.spacings.big};
+  height: ${designSystem.dimensions.spacings.big};
+
+  svg {
+    width: 100%;
+    height: 100%;
+  }
+`;
+const InlineContainer = styled.div`
+  display: flex;
+  height: 100%;
+  > * + * {
+    margin-left: ${designSystem.dimensions.spacings.m};
+  }
+`;
+const StackContainer = styled.div`
+  flex-grow: 1;
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  > * + * {
+    margin-top: ${designSystem.dimensions.spacings.m};
+  }
+`;
```

Comment:
Hmm sorry but we should be able to use the spacing components from the uikit. There is no need to re-implement them.
Can you clarify the reasoning here?

Reply:
![Screenshot 2020-05-25 at 12 14 40](https://user-images.githubusercontent.com/8238495/82803810-76d71b80-9e81-11ea-9420-06766a2018b9.png)

For situations like in the image above, I needed to the link aligned to the bottom of the page. Using spacings the height and flex-grow needed to be styled, but I understand the ui-kit components are not meant to be styled from the last conversation I had with you, hence custom containers.

Reply:
@emmenko  if you think it makes sense and is realistic to bring the requirement into the UI kit I can live with doing this sort of alignment later and wait for a UI Kit release.  

Use case: "I want a StackContainer to fill the available height and then be able to mark which of its children should fill the remaining height".    Basically tagging a specific child flex-grow: 1.  

Off-topic but FYI: From my API design experiments I have another use case that is pretty inconvenient with the UI kit since it was never intended for fully responsive :  Pass a responsive breakpoint to an inline stack to tell it from when on to wrap into one column (or just to generally wrap)

Reply:
I see, for this use case then it’s fine yes.
I thought you only wanted to define the spacing.

All good then 👌